app/controllers/posts_controller.rb

```diff
@@ -328,6 +328,7 @@ def preview
     @post ||= Post.new(user: current_user)
     @post.assign_attributes(permitted_params(false))
     @post.board ||= Board.find_by_id(3)
+    @post.editor_mode ||= current_user.default_editor
```

Comment:
Shouldn't this be set by permitted_params? Am I missing something there?

Reply:
Oh it's ||=, I'm not sur ehow you'd wind up with it not set other than it the immediate move over but fair enough

Reply:
Yeah! Just for the immediate move over, exactly - I think after that we should be fine to remove it here.

Reply:
Could remove it already tbh, it's not super important

```suggestion
```